backend/apps/ai_tutor/services.py

```
@@ -85,6 +87,68 @@
+                    temperature=0.7,
+                    stream=True,
+                )
+
+                for chunk in stream:
+                    if chunk.choices and len(chunk.choices) > 0:
+                        delta = chunk.choices[0].delta
+                        content = getattr(delta, "content", None)
+                        if content:
+                            yield f"data: {json.dumps({'text': content})}\n\n"
+                return
+            except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)